Add usual search / naming to disable placeholder selection (#12398)
filamentphp/filament: packages/tables/docs/04-filters/02-select.md

## Disable placeholder selection

> You can prevent the placeholder (null option) also known as the (Select All) option from being selected using the `selectablePlaceholder()` method:

Reviewer: ```suggestion
You can prevent the placeholder (null option with default label "All") from being selected using the `selectablePlaceholder()` method:
```

> You can prevent the placeholder (null option) also known as the (Select All) option from being selected using the `selectablePlaceholder()` method:

Reviewer 2: ```suggestion
You can remove the placeholder (null option), which disables the filter so all options are applied, using the `selectablePlaceholder()` method:
```
